storybook/.storybook/main.js

```diff
@@ -0,0 +1,44 @@
+const CopyWebpackPlugin = require("copy-webpack-plugin")
+const StyledComponentsTransformerPlugin = require("typescript-plugin-styled-components")
+const createStyledComponentsTransformer =
+  StyledComponentsTransformerPlugin.default
+module.exports = {
+  stories: ["../stories/**/*.tsx"],
+  addons: ["@storybook/addon-actions", "@storybook/addon-links"],
+  webpackFinal: (config) => {
+    const CopyWebpackPluginConfig = new CopyWebpackPlugin(
+      [
+        { from: "./node_modules/@misk/common/lib/web/" },
+        { from: "./node_modules/@misk/core/lib/web/" },
+        { from: "./node_modules/@misk/simpleredux/lib/web/" }
+      ],
+      { copyUnmodified: true }
+    )
+    const StyledComponentsTransformer = createStyledComponentsTransformer()
```

Comment:
We probably don't need this since Misk-Web has migrated to Emotion?

Reply:
I've removed it